packages/fluentui/react-northstar/test/utils/sharedIsConformant.tsx

```diff
@@ -0,0 +1,15 @@
+import { isConformant, IsConformantOptions } from '@fluentui/react-conformance';
+import { mountWithProvider } from 'test/utils';
+import { Ref, RefFindNode } from '@fluentui/react-component-ref';
+import { FocusZone } from '@fluentui/react-bindings';
+
+export function sharedIsConformant(testInfo: Partial<IsConformantOptions>, testFilePath: string) {
+  const defaultConfig = {
```

Comment:
If you explicitly type this, that will potentially make it easier to find errors or handle refactorings in the future.
```suggestion
  const defaultConfig: Partial<IsConformantOptions> = {
```